assets/admin_themes/Sea_Green/sea_green.scss

```
@@ -74,6 +74,7 @@
 @import "buttons/help-button-dropup.scss";
 @import "modal/modal.scss";
 @import "typography/typography.scss";
+@import "links/links.scss";
```

Stylelint (17.11.0)

[error] Unexpected extension ".scss" in `@import` (scss/load-partial-extension)

(scss/load-partial-extension)